extensions/interactions/GraphInput/directives/graph-viz.component.ts

```diff
@@ -197,6 +199,39 @@ export class GraphVizComponent implements OnInit, AfterViewInit {
     this.changeDetectorRef.detectChanges();
   }
 
+  handleKeyDown(event: KeyboardEvent): void {
+    if (this.state.currentMode === 2 && !this.isButtonOnTopOfDot()) {
+      const stepSize = 10;
```

Comment:
Names of variables measuring distances/etc. should have units.

Reply:
How about -- "stepSizeInPx" ?

Reply:
Yup that is fine.

Reply:
Done! PTAL